test-ts/end-to-end-vote-proposal.test.ts

```diff
@@ -0,0 +1,275 @@
+    await activateAndVoteTest();
+
+    await distributeEpochRewards(groups[0].address, rewardsGroup0.toString());
+    await distributeEpochRewards(groups[1].address, rewardsGroup1.toString());
+    await distributeEpochRewards(groups[2].address, rewardsGroup2.toString());
+
+    const minDeposit = await governanceWrapper.minDeposit();
+
+    const ownertx: ProposalTransaction = {
+      value: "0",
+      to: managerContract.address,
+      input: managerContract.interface.encodeFunctionData("owner"),
+    };
+
+    const ownertx2: ProposalTransaction = {
+      value: "0",
+      to: accountContract.address,
+      input: accountContract.interface.encodeFunctionData("owner"),
+    };
+
+    const proposal: Proposal = [ownertx];
+    const proposal2: Proposal = [ownertx2];
+
+    const dequeueFrequency = await governanceWrapper.dequeueFrequency();
+
+    await timeTravel(dequeueFrequency.toNumber() + 1);
+
+    const tx = await governanceWrapper.propose(proposal, "http://www.descriptionUrl.com");
+    await tx.send({ from: depositor1.address, value: minDeposit.toString() });
+
+    await timeTravel(dequeueFrequency.toNumber() + 1);
+
+    const tx2 = await governanceWrapper.propose(proposal2, "http://www.descriptionUrl2.com");
+    await tx2.send({ from: depositor1.address, value: minDeposit.toString() });
+
+    await timeTravel(dequeueFrequency.toNumber() + 1);
+
+    const tx3 = await governanceWrapper.propose(proposal2, "http://www.descriptionUrl2.com");
+    await tx3.send({ from: depositor1.address, value: minDeposit.toString() });
+
+    await timeTravel(dequeueFrequency.toNumber() + 1);
+    const dequeueProposalIfReadyTx = await governanceWrapper.dequeueProposalsIfReady();
+    await dequeueProposalIfReadyTx.send({ from: depositor1.address });
+
+    const proposalId = 1;
+    const index = 0;
+    const proposalId2 = 2;
+    const index2 = 1;
+
+    const depositor0NonVotedVotes = 100000;
+
+    const depositor0VotingPower = await voteContract.getVoteWeight(depositor0.address);
+    const depositor0VotedWeight = depositor0VotingPower.sub(depositor0NonVotedVotes);
+    const depositor1VotingPower = await voteContract.getVoteWeight(depositor1.address);
+
+    const voteProposalTx = await managerContract
+      .connect(depositor1)
+      .functions.voteProposal(proposalId, index, depositor1VotingPower, 0, 0);
+    await voteProposalTx.wait();
+
+    const voteProposal2Tx = await managerContract
+      .connect(depositor1)
+      .functions.voteProposal(proposalId2, index2, depositor1VotingPower, 0, 0);
+    await voteProposal2Tx.wait();
+
+    const voteProposal2Depositor0Tx = await managerContract
+      .connect(depositor0)
+      .functions.voteProposal(proposalId2, index2, depositor0VotedWeight, 0, 0);
+    await voteProposal2Depositor0Tx.wait();
+
+    const depositor1StakedCeloBalanceAfterVoting = await stakedCeloContract.balanceOf(
+      depositor1.address
+    );
+    expect(depositor1StakedCeloBalanceAfterVoting).to.eq(0);
```

Comment:
why this balance should drop to zero?

Reply:
This was a design choice - once StCelo is locked it will not appear in your wallet